src/SFA.DAS.Assessor.Functions/Domain/Print/DeliveryNotificationCommand.cs

```diff
@@ -41,6 +40,8 @@ public class DeliveryNotificationCommand : NotificationCommand, IDeliveryNotific
             _internalFileTransferClient.ContainerName = _settings.DeliveryNotificationInternalBlobContainer;
         }
 
+        public ICollector<string> StorageQueue { get; set; }
```

Comment:
Is there a better way of doing this. I hate having an optional property setter for something that really isn't optional. I.E it will break if the caller forgets to set it in the code but no way to catch it before runtime.

Reply:
Will add the queue as a parameter of the command